src/e2e/specs/landing.spec.ts

```diff
@@ -62,11 +66,10 @@ test.describe("Landing Page element verification", () => {
     });
 
     // visually verify landing page footer is displayed correctly
-    await expect(async () => {
-      await expect(landingPage.landingFooter).toHaveScreenshot(
-        `${process.env.E2E_TEST_ENV}-landingFooter.png`
-      );
-    }).toPass();
+    await expect(landingPage.landingFooter).toHaveScreenshot(
+      `${process.env.E2E_TEST_ENV}-landingFooter.png`,
```

Comment:
I think I only see `stage-*` baselines below, do we also need `dev-` and `prod-` baselines?
Which is interesting&hellip; so for every UI test, we'd need 3 envs &times; 2 OSes [per browser]. Or are we only expecting a subset of these tests to be run in production or local?

Reply:
great eye, will add prod env baselines, the ui regression will not be ran for dev, 

idea for uiregression:

-  is stage for daily cron/regression for ff/chrome and safari when that is added
- prod for on demand behavior verifications